diff --git a/setup.py b/setup.py
index 578d77a..06cbeee 100644
--- a/setup.py
+++ b/setup.py
@@ -1,7 +1,4 @@
-"""
-Py-Tree-sitter
-"""
-
+from datetime import datetime
 from os import path
 from platform import system
 from setuptools import Extension, setup
@@ -11,17 +8,17 @@ with open(path.join(path.dirname(__file__), "README.md")) as f:
     LONG_DESCRIPTION = f.read()
 
 setup(
-    name="tree_sitter",
-    version="0.20.0",
-    maintainer="Max Brunsfeld",
-    maintainer_email="maxbrunsfeld@gmail.com",
+    name="tree-sitter-builds",
+    version=datetime.utcnow().strftime("%Y-%m-%d"),  # Only used while building wheels
+    maintainer="Akuli",
+    maintainer_email="akuviljanen17@gmail.com",
     author="Max Brunsfeld",
     author_email="maxbrunsfeld@gmail.com",
-    url="https://github.com/tree-sitter/py-tree-sitter",
+    url="https://github.com/Akuli/py-tree-sitter-builds",
     license="MIT",
     platforms=["any"],
-    python_requires=">=3.3",
-    description="Python bindings to the Tree-sitter parsing library",
+    python_requires=">=3.6",
+    description="Binary wheels for using py-tree-sitter without a C compiler",
     long_description=LONG_DESCRIPTION,
     long_description_content_type="text/markdown",
     classifiers=[
@@ -40,5 +37,5 @@ setup(
             ),
         )
     ],
-    project_urls={"Source": "https://github.com/tree-sitter/py-tree-sitter"},
+    project_urls={"Source": "https://github.com/Akuli/py-tree-sitter-builds"},
 )
